### PR TITLE
Fix broken link in known issues

### PR DESCRIPTION
**Pull request by lepistone**

### CODE REVIEW

File: site/content/docs/user/known-issues.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ description: |-
 * [Pod Errors Due to "too many open files"](#pod-errors-due-to-too-many-open-files) (likely [inotify] limits which are not namespaced)
 * [Docker Permission Denied](#docker-permission-denied) (ensure you have permission to use docker)
 * [Windows Containers](#windows-containers) (unsupported / infeasible)
-* [Non-AMD64 Architectures](#non-amd64-architectures) (images not pre-built yet)
+* [Non-AMD64 Architectures](#nonamd64-architectures) (images not pre-built yet)
```
Comment (neolit123) [inline site/content/docs/user/known-issues.md:35]: the anchor is indeed  nonamd64...`, but unclear to me why that is (i.e. what is generating it without the `-`)

```
<h2 id="nonamd64-architectures">Non-AMD64 Architectures<a href="#nonamd64-architectures" class="hanchor" arialabel="Anchor"> 🔗︎</a> </h2>
```

https://kind.sigs.k8s.io/docs/user/known-issues/#non-amd64-architectures

/lgtm
/ok-to-test

**Comment by lepistone** [comment]
I re-authorised my corporate CLA and EasyCLA seems to be happy now, let's see if cla/linuxfoundation is also happy.

**Comment by lepistone** [comment]
I signed it

File: site/content/docs/user/known-issues.md (hunk 1 above)
Comment (BenTheElder) [reply site/content/docs/user/known-issues.md:35]: hugo generates the element IDs

**Comment by BenTheElder** [comment]
thanks!
/check-cla

**Comment by BenTheElder** [comment]
EasyCLA is passing, I'm manually overriding the robot and merging this.
Sorry for the hassle :(

Thank you for the fix!

**Pull request merged**